src/app/core/advertiser/advertiser.service.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable } from '@angular/core';
```

Comment:
whitespace

Reply:
Gotta get me one of them fancy lint-on-save editors.  I actually did finally get eslint autofixing in Atom.  Just haven't tried a typescript project yet.

Reply:
prettier can also be annoying, but it does keep the whitespace consistent